etc/zabbix/zabbix_agentd.d/ic_azure.conf

```diff
@@ -1,5 +1,5 @@
-# Microsoft Insights metric retrieval (configuration file, resource group, resource, timeshift, metric, statistic, timegrain)
-UserParameter=azure.insights.metric.timeshift[*],azure_insights_metric -c "$1" -g "$2" -r "$3" --timeshift "$7" "$4" "$5" "$6"
+# Azure metric retrieval (configuration file, resource group, resource, timeshift, metric, statistic, timegrain)
```

Comment:
New parameters are missing from list.

Reply:
I don't understand. Everything seems to be committed here:
https://github.com/digiaiiris/zabbix-azure-monitoring/blob/e0c32e3c1474da2d455d8e8592bc23d995c1cfea/etc/zabbix/zabbix_agentd.d/ic_azure.conf

Reply:
This was fixed in commit aa4284da257ce6fcb305b2bd90ebe741af84c58b.